index.css

```diff
@@ -537,37 +562,38 @@ body {
   /* no unit */
   top: calc(50% - 0px / 2 + 0.5px);
   border: 1px solid #fff;
-  margin: 120px 24px 0 24px;
+  margin: 10px 5% 0 5%;
 
   /* no unit */
 }
 
 .multi_post {
-  width: 100%;
+  width: 40%;
 }
```

Comment:
- Kindly, stick to the Figma design as much as possible, the description of your recent work is centered make some Figma as your previous code reviewer suggested.👌👍



| Your design                                                             | Figma design                                         |
| --------------------------------------------------------------------------------| --------------------------------------------------------- |
|<img src="https://user-images.githubusercontent.com/52010789/206779620-819d445f-a476-4640-8ccd-a967c3715efa.png" width="200">|<img src="https://user-images.githubusercontent.com/52010789/206779648-334db76a-cc10-45c3-9df4-aab3fc525096.png" width="200">|


😎️